src/OrchardCore.Mvc.Web/Program.cs

```diff
@@ -12,7 +12,6 @@ public static Task Main(string[] args)
         public static IHost BuildHost(string[] args) =>
             Host.CreateDefaultBuilder(args)
                 .ConfigureWebHostDefaults(webBuilder =>webBuilder.UseStartup<Startup>())
-                .Build()
-            ;
+                .Build();
```

Comment:
Again the format for this line `=>webBuilder.UseStartup<Startup>())` is in proper as well as the spacing the above one is look good

Reply:
LGTM